code/modules/mob/living/simple_animal/friendly/butterfly.dm

```diff
@@ -14,7 +14,9 @@
 	health = 2
 	harm_intent_damage = 1
 	friendly = "nudges"
-	pass_flags = PASSTABLE
+	density = 0
```

Comment:
This was needed, if cargo ever unleashed a butterfly box in an open place, they would quickly get very annoying.